screenshot.js

```diff
@@ -0,0 +1,97 @@
+const toughCookie = require('tough-cookie');
+const fileUrl = require('file-url');
+const isUrl = require('is-url-superb');
+
+module.exports = class Screenshot {
```

Comment:
Does this class makes sense? I did it so that I wouldn't have to pass around `page` everywhere if I kept it functional.